.github/workflows/build-end-to-end-test.yaml

```diff
@@ -1,51 +1,146 @@
 name: end-to-end-tests
-
 on:
   workflow_call:
     inputs:
       environment:
         type: string
         description: Environment
         required: true
+      test:
+        type: string
+        description: Environment
+        required: true
 
```

Comment:
please add job-init to every job, similar to other repositories so we can use github lib and slack notifications